crazyflie/scripts/crazyflie_server.py

```diff
@@ -534,10 +516,11 @@ def _log_custom_data_callback(self, timestamp, data, logconf, uri):
         for log_name in data:
             log_type = self.swarm._cfs[uri].l_toc[log_name.split(
                 '.')[0]][log_name.split('.')[1]].ctype
-            msg = cf_log_to_ros_topic[log_type]()
-            msg.data = data.get(log_name)
-            self.swarm._cfs[uri].logging["custom_log_publisher"][log_name].publish(
-                msg)
+            msg = LogDataGeneric() 
```

Comment:
Would be good to also set the header (with ROS timestamp) and STM timestamp

Reply:
ahhh yes! I also figured this out since in rqt_plot it wouldn't plot it properly due to that.